src/plugins/kibana_usage_collection/server/collectors/ops_stats/ops_stats_collector.ts

```diff
@@ -51,6 +50,22 @@ export function getOpsStatsCollector(
     type: KIBANA_STATS_TYPE,
     isReady: () => !!lastMetrics,
     fetch: () => lastMetrics,
+    schema: {
```

Comment:
We didn't have a schema for the collector!

Reply:
Yeah, this is a `StatsCollector`, so it's not mandatory to declare them.